### ISSUE TITLE
After upgrade from 5.8.x to 5.10.x, searching invoices no longer works

### ISSUE THREAD

**Issue by link470** [invoiceninja/ui#1973]
Previously, I was able to search for a specific item across all invoices.  For example, "cable", and every invoice showing cable either in the product or the description would show up.  Now, searching for key words doesn't seem to do anything in the React UI.

Can the ability to search by item/description key words please be added back to the React UI?

**Comment by beganovich** [comment]
Hey there. Can you check the console (network tab) to see if any requests fire? Or if there's an error in the console.

Thanks.

**Comment by link470** [comment]
Using the Search box in the top menu bar only seems to search for clients, as that's the only thing that seems to autocomplete.  

I think I've discovered what the problem is for the invoice filter.  If I navigate to Invoices and use the _Filter_ box at the top of the list, I was able to get some search results.  The search results only seem to be searching the _Description_ field, but not any item names on the invoice.  So for an example, for an item called _6ft Cable_ and a description of _Black power cable_, if I search for "power" or "cable", the invoice containing that item appears in the search results.  If I search for "6ft", I get no results.

It would be great to have the Item field searched as well.
